posthog/api/exports.py

```diff
@@ -114,6 +120,15 @@ def create(self, validated_data: Dict, *args: Any, **kwargs: Any) -> ExportedAss
         return instance
 
 
+def content_for_asset(instance: ExportedAsset):
+    if instance.export_format != "text/csv":
+        return instance.content
+
+    storage_location = instance.export_context.get("storage_location", None)
+    content_bytes = object_storage.read_bytes(storage_location)
+    return gzip.decompress(content_bytes)
```

Comment:
What's the benefit of lifting it to be a field? Either in the context JSON or as a field on the table it's either there or not.

Hmm, you could ask the same question flipped on its head... It's not very discoverable hidden in the blob 👍

Reply:
I guess my thinking is having a generic JSONField makes that field quite "random". Arguably it should even be immutable so that we know exactly what parameters were used for the export.

Reply:
Yep, I secretly hate explicit schema databases but this is a good field to put front and center :)